setup.cfg

```diff
@@ -31,3 +31,5 @@ python_requires = >=3.8
 voice =
     opuslib
     pynacl
+testing =
```

Comment:
Maybe a workflows name?

Reply:
i mean it's gotta be  used for testing in general sooo

Reply:
I mean in my opinion it looks like it's just a dummy name we made up for a quick test. Maybe formatters?

Reply:
> ust a dummy name we

Well testing will regroup the nox suite & pytest, for workflows and user-side testing
Cant really think of a better name

Reply:
Formatting, Workflows, Checks, Actions